### ISSUE TITLE
A11y_ Windows Template Studio_ MVVM Basic Details_Screenreader: Screenreader is not announcing the link “The Model-View View Model pattern”.

### ISSUE THREAD

**Issue by Yash14j** [microsoft/WindowsTemplateStudio#3897]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#GH_WindowsTemplateStudio_Oct2020;#Windows Template Studio;#A11y_Windows Template Studio;#Win10;#FTP;#STP;#DesktopApp;#WCAG1.3.1;#NVDA;#Narrator;#Screenreader;#NonBenchmark;

### **Environment Details:**
Application Name: Windows Template Studio (WinTS)
Templates version: 3.8.20273.1
Microsoft Visual Studio Enterprise 2019 Preview
Version:16.8.0 Preview 3.1 
Windows Version: Windows10
NVDA Version 2019.3.1

### **Repro Steps:**

1. Open Visual Studio 2019 
2. Navigate to File from menu with the help of tab key and press enter key to expand it. 
3. Select new, press enter to expand and then select project.
4. Select the "Windows template studio (WPF.Net Core)". Navigate to next button with the help of tab key and press enter key
5. "Configure your new project" screen will appear. 
6. Navigate to create button with the help of tab key and press enter key.
7. New WPF App screen will open.
8. Navigate to Design pattern in the left navigation menu with the help of tab key and click on it.
9. Select a design pattern screen will appear. Tab Navigate to MVVM Basic and click on details.
10. Open the screenreader and navigate to link “The Model-View View Model pattern” with the help of tab key to check the result.

### **Actual Result:**
When we navigate to the link “The Model-View View Model pattern” with the help of tab key, screenreader does not announce anything.

### **Note:**
This issue is observed with both NVDA and NARRATOR.
This issue is observed in following scenarios:
•	MVVM Light Details
•	Prism Details
•	Web View details
•	Optional Login Details
•	XAML Island Details
•	MSIX Packaging Details
•	Forced Login Details

### **Expected Result:**
Sreenreader should announce the name of the link “The Model-View View Model pattern” when screenreader user navigates to it with the help of tab key.

### **User Impact:**
Screenreader user will not get to know about the link “The Model-View View Model pattern”.

**Comment by sibille** [comment]
Verified in dev-nightly version:
Wizard version: 0.22.20310.1

You can install the dev-nightly version from our nightly feed (https://www.myget.org/F/windows-template-studio-nightly/vsix/).
For instructions on how install WTS from the dev-nightly feed in Visual Studio see https://github.com/microsoft/WindowsTemplateStudio/blob/dev/docs/getting-started-extension.md#nightly--pre-release-instructions

**Comment by Yash14j** [comment]
Issue does not repro in the Wizard version: 0.22.20310.1 hence closing this bug.
